spec/system/management/account_spec.rb

```diff
@@ -102,4 +102,26 @@
     expect(page).to have_css("a[href='javascript:window.print();']", text: "Print password")
     expect(page).to have_css("div.for-print-only", text: "another_new_password", visible: :hidden)
   end
+
+  describe "When a user has not been selected" do
+    before do
+      Setting["feature.user.skip_verification"] = "true"
+    end
+
+    scenario "we can not reset password via email" do
```

Comment:
Typo:

```suggestion
    scenario "we cannot reset password via email" do
```

Or:

```suggestion
    scenario "we can't reset password via email" do
```

Same in the other tests.